src/main/java/de/tum/in/www1/artemis/domain/metis/ConversationParticipant.java

```diff
@@ -39,6 +39,9 @@ public class ConversationParticipant extends DomainObject {
     @Column(name = "is_hidden")
     private Boolean isHidden;
 
+    @Column(name = "is_muted")
+    private Boolean isMuted;
```

Comment:
Would be great if you could turn this into a primitive boolean and adjust other places in the code accordingly.